packages/common/src/sandbox/modules.ts

```diff
@@ -284,3 +301,19 @@ export const findCurrentModule = (
     mainModule
   );
 };
+
+export const resolveModuleWrapped = sandbox => (path: string) => {
```

Comment:
Moved these 2 utils from `templates/configuration/resolve-module-wrapped.ts`, needed to use one of them. This feels like a nicer home for them.